### ISSUE TITLE
Remote sms link not working

### ISSUE THREAD

**Issue by glitch666** [ianmarmour/nvidia-clerk#160]
Now that nvidias api is back up, when I test with the -sms and -remote options the text I receive on my phone is incomplete. I am testing with the 2060 model.
![Screenshot_20200930-142127](https://user-images.githubusercontent.com/47763234/94741540-1f622400-0329-11eb-8b49-9058e486f10e.png)

**Comment by redstone2019** [comment]
yep we are having some issues because Nvidia rolled out API changes that is one of them

**Comment by redstone2019** [comment]
Do you have the latest version (3.8.2)? @glitch666

**Comment by glitch666** [comment]
> Do you have the latest version (3.8.2)? @glitch666

Yes!

**Comment by redstone2019** [comment]
ok

**Comment by redstone2019** [comment]
Yeah, that is an issue that will be addressed sometime soon.
